contrib/test.sh

```diff
@@ -67,4 +70,7 @@ then
     fi
 
     cargo test --verbose
+
+    cd ..
+    rm -rf dep_test
```

Comment:
Let's hope no one ever links `dep_test` to `/` :stuck_out_tongue_winking_eye:

Reply:
If they do, the script will fail before, on `cargo new dep_test`